torchci/lib/bot/pytorchBotHandler.ts

```diff
@@ -317,9 +317,12 @@ The explanation needs to be clear on why this is needed. Here are some good exam
           args.rebase
         );
       case "rebase": {
-        if (args.stable) {
+        if (!args.branch) {
```

Comment:
Hmm, this will break the current behavior of `rebase -s`. This should be `if (arg.stable or !args.branch)` I think. Or do you think we should remove `rebase -s` now this is going to be the default behavior.

Reply:
How would this break `rebase -s`? Wouldn't it preserve the existing behavior since that also rebases onto viable/strict?

Reply:
Oh, I see. No branch would be set when `rebase -s` is used because `-s`, `-m`, and `-b` are mutually exclusive.  May be we could consider removing `rebase -s` because it doesn't do anything now, or keep it just in case someone out there still uses it, but with a OBSOLETED warning text in the help message. Thoughts?

Reply:
Sure, we could add an obsolete text. And yes, I deliberately left `-s` in for the folks who currently use it (and some folks do)